wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -1,406 +1,355 @@
-<tr class="header">
-<th><p>Button<br />
-(Keyboard Shortcut)</p></th>
-<th><p>Usage</p></th>
-<th><p>Description</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>New Combo<br />
-(q)</p></td>
-<td><p><strong>Right-click:</strong> Change current note to next combo colour.</p></td>
-<td><p>Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20.</p></td>
-</tr>
-<tr class="even">
-<td><p>Hit Sounds</p></td>
-</tr>
-<tr class="odd">
-<td><p>Whistle<br />
-(w)</p></td>
-<td><p>(&quot;Select&quot; on a Hit note) <strong>Left-Click:</strong> Use whistle sound on this note</p></td>
-<td><p>Add a whistle sound addition to the selection.</p></td>
-</tr>
-<tr class="even">
-<td><p>Finish<br />
-(e)</p></td>
-<td><p>(&quot;Select&quot; on a Hit note) <strong>Left-Click:</strong> Use finish sound on this note</p></td>
-<td><p>Add a finish (cymbal) sound addition to the selection.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Clap<br />
-(r)</p></td>
-<td><p>(&quot;Select&quot; on a Hit note) <strong>Left-Click:</strong> Use clap sound on this note</p></td>
-<td><p>Add a clap sound addition to the selection.</p></td>
-</tr>
-<tr class="even">
-<td><p>Assist Tools</p></td>
-</tr>
-<tr class="odd">
-<td><p>Grid Snap<br />
-(t)</p></td>
-<td><p><strong>Shift (Hold):</strong> Temporary toggle.<br />
-<strong>Ctrl (Hold):</strong> Temporary disable angle snapping.</p></td>
-<td><p>Hit notes will be snapped to the grid by default.</p></td>
-</tr>
-<tr class="even">
-<td><p>Distance Snap<br />
-(y)</p></td>
-<td><p><strong>Alt (Hold):</strong> Temporary toggle. Switch Beat Snap Divisor to Distance Snap<br />
-<strong>Alt + Mousewheel:</strong> Adjust Distance Snap multiplier.</p></td>
-<td><p>Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Lock Notes<br />
-(l)</p></td>
-<td><p>(&quot;Select&quot; on a Hit note) <strong>Left-Click:</strong> Lock this note</p></td>
-<td><p>Lock selected note to current position and timestamp.</p></td>
-</tr>
-</tbody>
-</table>
-
-Bottom (Song's Timeline)
-------------------------
-
-<img src="BE_STL.jpg" title="fig:Song&#39;s Timeline" alt="Song&#39;s Timeline" width="200" /> At **bottom-left**, it shows **timestamp in milliseconds (ms)** and **song duration %**. The "%" may be switched to "intro/outro" when there is intro/outro storyboarding.
+### Centre-Right (Hit Sounds & Assist Tools)
+
+**List of available subtools complement to the original DIY toolkit.**
+
+![Hit Sounds & Assist Tools](BE_HSAT.jpg "Hit Sounds & Assist Tools")
+
+| Button (Keyboard Shortcut) | Usage | Description |
+| -------------------------- | ----- | ----------- |
+| New Combo (q) | **Right-click:** Change current note to next combo colour. | Note combo. This is equivalent to a verse / measure in song / instrument script. An acceptable range is about 5-20. |
```

Comment:
since this is a keyboard key, change `(q)` to ``(`Q`)``
please do this for any of the other tables below as well